frontend/src/App.js

```diff
@@ -1,13 +1,48 @@
-import React from "react";
+import React, { Component } from "react";
 import ReactDOM from "react-dom";
 import HelloWorld from "./HelloWorld";
 
-const App = () => {
-  return (
-    <div>
-      <HelloWorld text="Hello World" />
-    </div>
-  );
-};
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      list: []
+    }
+  }
 
-ReactDOM.render(<App />, document.getElementById("root"));
+  componentDidMount() {
+    fetch("/api/getSomething")
```

Comment:
I should extract that to some function, but since this is only a sample file I decided to commit this way.